signer/src/transaction_signer.rs

```diff
@@ -657,6 +650,14 @@ where
         Ok(())
     }
 
+    async fn check_blocklist(&self, address: &str) -> bool {
```

Comment:
nit: the name can be confusing wrt the return value (eg, does true mean the blocklist wanna block this?); what about something like `(check_)?blocklist_accept` or similar?

Reply:
Good point, will update.

Reply:
Done in https://github.com/stacks-network/sbtc/pull/589/commits/59c41005ab2e6dca751f18aa1ea960129e28acc9.